lib/resque/pool.rb

```diff
@@ -344,7 +344,42 @@ def quit_excess_workers_for(queues)
     end
 
     def worker_delta_for(queues)
-      config.fetch(queues, 0) - workers.fetch(queues, []).size
+      config_get_worker_count(queues).to_i - workers.fetch(queues, []).size
+    end
+
+    # Get the number of workers declared in the config file for queues
+    #
+    # @param queues [String] names of queues separated by comma if many exist
```

Comment:
Since we're on it, let's document the return values too.

Reply:
Fixed!